src/dataHandler.py

```diff
@@ -59,11 +62,12 @@ def print_to_console(data: bytes or str):
 """
 
 
-def print_to_console_json(data: bytes or str):
+def print_to_console_json(data: bytes or str, timestamp_format=""):
     if type(data) is bytes:
         data = data.decode('utf-8').strip()
     if data:
-        output = {'payload': data}
+        timestamp = datetime.now().strftime(timestamp_format)
+        output = {'payload': f"{timestamp}{data}"}
```

Comment:
This could possibly introduce a "corruption" of the data that is being sent by the user. Meaning, depending on the way that the host (in this case, the vscode-arduino extension) is attempting to use the data, adding the timestamp to the payload itself would cause issues. 
To avoid this, pass the timestamp as another field of the json object being sent. This also would allow for the host to decide whether they want to use the timestamp or not. 
Note: Requires changes in host code as well.

Reply:
Changed in https://github.com/microsoft/serial-monitor-cli/pull/10/commits/ca6c4131991c285f4f43ec6315565169692dc577. Output with JSON enabled now looks like this:
```sh
❯ python3 src/main.py open -b 9600 /dev/cu.usbmodemFA131 -t "%H:%M:%S.%f -> " -j  
{"payload": "0", "timestamp": "09:27:51.511835 -> "}{"payload": "1", "timestamp": "09:27:52.511187 -> "}{"payload": "2", "timestamp": "09:27:53.510583 -> "}{"payload": "3", "timestamp": "09:27:54.513876 -> "}{"payload": "4", "timestamp": "09:27:55.513435 -> "}{"payload": "5", "timestamp": "09:27:56.512731 -> "}
...
```